UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1010,14 +1010,15 @@ public void HealDamageOnAll(GameObject healingItem, float healAmt, DamageType da
 				}
 			}
 
-			SurfaceReagents[bodyPartAim].Add(Chemicals);
+			//Does not trigger when bodyPartAim is none, intended or should be random?
+			if(SurfaceReagents.TryGetValue(bodyPartAim, out var mix) == false) return;
```

Comment:
Maybe it could alternatively apply to all? CC @Bod9001 thoughts?

Reply:
yeah if it is none that should apply to all since, there would be scenarios where you'd be getting splashed from scripts like smoke and stuff

Reply:
Done